server/server.go

```diff
@@ -386,7 +404,8 @@ func listenAndServe(wg *sync.WaitGroup, port string) {
 	}
 }
 
-// StartServer starts service at given port.
+// StartServer invokes functions to start http server at given port.
```

Comment:
"StartServer sets up the different http endpoints to be used by the client and starts the http server at the given port.".

Reply:
Done